### ISSUE TITLE
pdf does not print instructions after int3 

### ISSUE THREAD

**Issue by wavvs** [radare/radare2#9649]
### Expected behavior
I assume that `pdf` (pdc, pdr, maybe other?) should disassemble whole function, but it can't  disassemble after **int3** instruction. `pd` works as expected.
### Actual behavior
Recorded behavior: https://asciinema.org/a/zCQiST74pLRckCyWcrCOSuisT

### Steps to reproduce the behavior 
- Use this binary [lab2.zip](https://github.com/radare/radare2/files/1799910/lab2.zip)

**Comment by radare** [comment]
Int3 is a breakpoint, no execution will happen after that. Why it should be considered differently? And pdf, pdr, pdc and others are completely unrelated to the code analysis loops

**Comment by wavvs** [comment]
I expect that if `pdf` disas whole function it shouldn't stop at int3, it should disas until last ret instruction, right?

**Comment by radare** [comment]
Pdf shows what the previous analysis did. So its not pdf fault. And no, not all functions end with a ret. some do with jmp others with call , others with breakpoint instructions etc

**Comment by wavvs** [comment]
Yeah, my mistake, sorry, I meant function in my situation. So it is not an issue or something with analysis?

**Comment by radare** [comment]
I didnt had time to check this bin with my laptop yet. But one option to skip that would be to add anal hints or tweak the analysis loop or the plugin to make int3 instructions be considered a nop. But anyway i doubt any anyalisis tool will consider an int3 not the end of a function
